tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -527,11 +596,18 @@ def test_lm_featurizer_number_of_sub_tokens_process(
     [(20, "bert", False), (1000, "bert", True), (1000, "xlnet", False)],
 )
 def test_sequence_length_overflow_train(
-    input_sequence_length: int, model_name: Text, should_overflow: bool
+    input_sequence_length: int,
+    model_name: Text,
+    should_overflow: bool,
+    create_language_model_featurizer: Callable[
+        [Dict[Text, Any]], LanguageModelFeaturizerGraphComponent
+    ],
+    monkeypatch: MonkeyPatch,
 ):
-    component = LanguageModelFeaturizer(
-        {"model_name": model_name}, skip_model_load=True
+    monkeypatch.setattr(
```

Comment:
Way better than having production code changed for `pytest` 🚀

Reply:
one thing: should we move the monkeypatching into the fixture to avoid repeating this?